Assignment/Program.cs

```diff
@@ -1,7 +1,20 @@
-﻿namespace Assignment {
-   internal class Program {
-      static void Main (string[] args) {
-         Console.WriteLine ("Hello world!");
+﻿while (true) {
+   Console.Write ("Give number of rows : ");
+   if (int.TryParse (Console.ReadLine (), out int n) && (n > 0)) {

```

Comment:
Same mistake - Try handling null or other values.

Reply:
Nullability is handled while passing "Console.ReadLine" as a parameter inside TryParse. 
Other values are also handled.